src/components/faceVerification/screens/IntroScreen.jsx

```diff
@@ -171,27 +193,31 @@ const IntroScreen = ({ styles, screenProps, navigation }) => {
     [isFVFlow, firstName, fullName],
   )
 
-  const [disposing, checkDisposalState] = useDisposingState(
-    {
-      requestOnMounted: false,
-      enrollmentIdentifier,
-      fvSigner,
-      onComplete: isDisposing => {
-        if (!isDisposing) {
-          return
-        }
-
-        const dialogData = showQueueDialog(WalletDeletedPopupText, true, {
-          onDismiss: isFVFlow ? () => fvRedirect(false, 'Wait 24 hours') : goToRoot,
-          imageSource: Wait24HourSVG,
-        })
-
-        showDialog(dialogData)
-      },
+  const onDeny = useCallback(
+    reason => {
+      return isFVFlow ? fvRedirect(false, reason) : goToRoot()
     },
-    [enrollmentIdentifier],
+    [isFVFlow],
   )
 
+  const [disposing, checkDisposalState] = useDisposingState({
```

Comment:
@sirpy If someone cannot delete their face-id anymore, why keep this disposing state?